src/code-generators/typescript-generators/namedImport.ts

```diff
@@ -53,26 +57,54 @@ export function generateNamedImport(
     const lib = `${stringQuoteStyle}${imp.libraryName}${stringQuoteStyle}${eol}`;
 
     const specifiers = imp.specifiers.sort(specifierSort).map(o => generateSymbolSpecifier(o)).join(', ');
-    let importSpecifiers = `${space}${specifiers}${space}`;
-    if (importSpecifiers.trim().length === 0) {
-        importSpecifiers = ' ';
-    }
-
-    const importString = importTemplate(
-        getImportSpecifiers(imp, spaceBraces),
-        lib,
-    );
+    let retVal:string = '';
 
-    if (importString.length > multiLineWrapThreshold) {
+    if (specifiers.length > multiLineWrapThreshold) {
```

Comment:
This actually changes the behaviour of the multi line wrap threshold. before, the generated string was checked for his length. Now only the specifiers.

Or did I misread something?

Reply:
It should behave like before, or we need to introduce a BREAKING CHANGE

Reply:
I actually intended this change, because I thought that it is the right thing to do, didn't realize that it could be considered a Breaking change.

Reply:
Actually, I will rever that part for now. Its small thing to change later if think that it should be done.